### PR TITLE
Added documentation for Vmware driver on Windows 

### LINKED ISSUE #5830: create documentation for vmware driver on Windows
our site currently only has vmware docs for mac os: https://minikube.sigs.k8s.io/docs/start/macos/

we need to add documents to the site for vmware on windows.
https://minikube.sigs.k8s.io/docs/start/windows/

### PR DESCRIPTION
**Pull request by Aut0R3V**
"fixes #5830 "

### CODE REVIEW

**Comment by bl-ue** [comment]
@Aut0R3V the bot (@k8s-ci-robot) doesn't like you to put the issue number in the PR title.  You should put it in the description instead.

**Comment by Aut0R3V** [comment]
Sure, I've changed it

**Comment by Aut0R3V** [comment]
@bl-ue can you please give an idea on how to resolve the incomplete checks

**Comment by bl-ue** [comment]
@Aut0R3V it looks like a maintainer like @afbjorklund or @tstromberg needs to write `/ok-to-test` so that testing can happen, the tide check...a maintainer needs to `/lgtm` and `/approve`.

/assign @medyagh

**Comment by Aut0R3V** [comment]
Oh I'm sorry I forgot that

**Review by bl-ue — APPROVED** [review]

File: site/content/en/docs/drivers/includes/vmware_windows_usage.inc
Comment (medyagh) [inline site/content/en/docs/drivers/includes/vmware_windows_usage.inc:12]: no need for 4GB and 4 Cpus options here.. minikube should work with Default Auto-select values. (it is better not to set it so minikube sets them for u automatically)

**Review by medyagh — CHANGES_REQUESTED** [review]

File: site/content/en/docs/drivers/includes/vmware_windows_usage.inc
Comment (Aut0R3V) [reply site/content/en/docs/drivers/includes/vmware_windows_usage.inc:12]: sure I'll edit that. Thanks!

File: site/content/en/docs/drivers/includes/vmware_windows_usage.inc
Comment (priyawadhwa) [inline site/content/en/docs/drivers/includes/vmware_windows_usage.inc:12]: we probably want `--driver vmware` here, because minikube will default to using docker if it's installed

**Review by priyawadhwa — COMMENTED** [review]
Thank you for adding this documentation, I've left one suggestion.

**Comment by Aut0R3V** [comment]
@priyawadhwa @medyagh can you guys please have a look and update if any other changes might be required?

File: site/content/en/docs/drivers/includes/vmware_windows_usage.inc
Comment (tstromberg) [inline site/content/en/docs/drivers/includes/vmware_windows_usage.inc:3]: There should be a space after the asterisk, to make this a bullet point. See: https://deploy-preview-9937--kubernetes-sigs-minikube.netlify.app/docs/drivers/vmware/

**Comment by tstromberg** [comment]
/lgtm 

NOTE: There is one space that is missing. Please add it to fix the bullet, and see https://deploy-preview-9937--kubernetes-sigs-minikube.netlify.app/docs/drivers/vmware/ to verify that it looks like you intend it to.

**Pull request merged**
